src/transformers/file_utils.py

```diff
@@ -89,10 +89,13 @@
             try:
                 _tf_version = importlib_metadata.version("tensorflow-cpu")
             except importlib_metadata.PackageNotFoundError:
-                _tf_version = None
-                _tf_available = False
+                try:
+                    _tf_version = importlib_metadata.version("tf-nightly")
+                except importlib_metadata.PackageNotFoundError:
+                    _tf_version = None
+                    _tf_available = False
     if _tf_available:
-        if version.parse(_tf_version) < version.parse("2"):
+        if version.parse(_tf_version) < version.parse("2.3"):
```

Comment:
No this was just an old check for version 1 that was left, but we should not hardcode anything here, as we will forget to update it when we pin tensorflow to another release.

Reply:
Should I let it as it is or completely remove this condition?

Reply:
I think we should leave it as is. Let's see what the others think!

Reply:
I agree with Sylvain that this should stay as it is, it's to indicate that the general framework detected (TensorFlow 1) will not work correctly as we expect another (TensorFlow 2).